src/cutscenes/reimu_good_end.inc.h

```diff
@@ -0,0 +1,77 @@
+
+.name = "Reimu Good Ending",
+.bgm = "ending",
+
+.phases = (CutscenePhase[]) {
+	{ "cutscenes/locations/moriya", {
+		T_NARRATOR("— The Moriya Shrine"),
+		T_NARRATOR("A rival shrine at the top of Yōkai Mountain.\n"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/01", {
+		T_REIMU("Listen, Sanae, I don’t know what a ‘speedrun’ is, and I don’t really care."),
+		T_SANAE("You’re such a stick in the mud, Reimu!"),
+		T_SANAE("You like resolving incidents as quickly as possible, right? It’s like that, but with video games!"),
+		T_REIMU("I don’t know anything about ‘video games’, either!"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/01", {
+		T_NARRATOR("The Tower had been shut down, ending the crisis of madness sweeping land."),
+		T_NARRATOR("Almost everyone had returned to normal, although a few lamented the return to status quo."),
+		T_NARRATOR("The instigators, Kurumi and Elly, had been conscripted into ‘community service’ by the Moriya Shrine Gods."),
+		T_NARRATOR("Although the bounds of that ‘community service’ seemed to expand as time went on…"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/02", {
+		T_KANAKO("I can’t believe it! Is nuclear fusion really that simple?!"),
+		T_ELLY("Y-Yes, it’s just a matter of changing that coefficient there, and shaping the magnetic containment field using-"),
+		T_KANAKO("Of course! I was such a fool! I can’t believe I ever bothered attempting to harness the Yatagarasu!"),
+		T_KANAKO("Finally, I have the final key to modernizing Gensokyo, and preventing the energy crisis!"),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/02", {
+		T_ELLY("O-oh dear… are you sure you’re well, ma’am?"),
+		T_SUWAKO("Heh heh, don’t worry, thisis as normal as it gets for Kanako."),
+		T_ELLY("‘Normal’? Really?"),
+		T_SUWAKO("Kanako is a green energy fanatic. Don’t you think she’s cute when she’s like this?"),
+		{ 0 },
+	}},
+	{ "cutscenes/locations/moriya", {
+		T_NARRATOR("Reimu was spending more time on Yōkai Mountain, claiming she was ‘keeping watch’."),
+		T_NARRATOR("For some reason, she seemed to have a difficult time settling down from this incident in particular."),
+		{ 0 },
+	}},
+	{ "cutscenes/reimu_good/03", {
+		T_REIMU("Hey, Elly."),
+		T_ELLY("Ah, yes, what is it?"),
+		T_REIMU("I’m still surprised you’re not still pissed off at everyone. Not even a little bit."),
+		T_REIMU("You were shouting like a dictator up there."),
+		T_ELLY("In the end, it seems the Tower got to me, too."),
+		T_ELLY("I’m still sad about my own Gensokyo, but to go that far…"),
+		{ 0 },
```

Comment:
See the definition of `CUTSCENE_PHASE_MAX_TEXT_ENTRIES` in `src/cutscenes/scene_impl.h`, and the corresponding comment. It's currently set to `6`, which as per documentation includes the `{ 0 }` terminator. You're exceeding that limit here. You must either rise that limit, or break up the phase (whichever looks better).

GCC and Clang should emit a warning about this. I recommend building with `-Dwerror=true -Ddeprecation_warnings=no-error` to catch such issues.